src/Toolkit/Toolkit.WPF/UI/Controls/FeatureForm/FeatureFormView.Theme.xaml

```diff
@@ -36,7 +36,7 @@
       <Setter.Value>
         <ControlTemplate TargetType="{x:Type primitives:DateTimePickerFormInputView}">
           <StackPanel>
-            <DatePicker x:Name="DatePicker" SelectedDate="{Binding Value, Mode=OneWay}" IsEnabled="{TemplateBinding IsEnabled}" />
+            <DatePicker x:Name="DatePicker" IsEnabled="{TemplateBinding IsEnabled}" />
```

Comment:
Why this change? Does DatePicker still initialize correctly?

Reply:
The control itself handles updating this - with the binding two things were updating, causing one edit to overwrite the other, preventing you from setting a date to null.